assets/section-email-signup-banner.css

```diff
@@ -0,0 +1,41 @@
+.email-signup-banner__box .email-signup-banner__heading {
+  margin-bottom: 0;
+}
+
+.email-signup-banner__box > * + .newsletter__subheading {
+  margin-top: 2rem;
+}
+
+.email-signup-banner__box .newsletter__subheading p {
+  margin: 0;
+}
+
+.email-signup-banner-background {
+  width: 100%;
+  height: 100%;
+  position: relative;
+  left: 50%;
+  transform: translateX(-50%);
+}
+
+.email-signup-banner-background-mobile {
+  position: relative;
```

Comment:
Are these styles needed? `email-signup-banner-background` already includes the same CSS properties.

Reply:
Good catch!